### ISSUE TITLE
Validation and Dropdown when writing addresses to search representatives

### ISSUE THREAD

**Issue by MikelTafalla** [bcantello/peoples_voice_project#61]
Right now we can input anything on the address fields. Not sure if this is something we want to do, but for city and state, we can probably add dropdowns with the cities and states of the US? or add some kind of validation on those fields?

**Comment by bcantello** [comment]
Yeah we could definitely turn the state into a dynamic search field. The rest would be a bit tough. There are too many cities and towns and whatever else for us to reliably be able to create this on our own for those fields. Address validation services are costly. I'm not sure how it works for address auto fill without validation. we would have to look into this a lot more
